setup.cfg

```diff
@@ -1,7 +1,7 @@
 # https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
 [metadata]
 name = h_pyramid_sentry
-version = 1.0
+version = 1.1
```

Comment:
Let's make sure our consumers don't get a nasty surprise if they were using us to get their dependencies. We know they aren't at this point, but good habit nonetheless.